### DISCUSSION TITLE
Dịch response

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#149] [Terminology] [answered]
Từ này xuất hiện trong cụm "local response normalization"
> AlexNet also uses a competitive normalization step immediately after the ReLU step of layers C1 and C3, called local response normalization (LRN): the most strongly activated neurons inhibit other neurons located at the same position in neighboring feature maps (such competitive activation has been observed in biological neurons).

Dịch trong context signal có thể là `phản ứng` hoặc `đáp ứng` (từ thứ 2 có vẻ hay được dùng hơn trong các tài liệu tiếng Việt, e.g. [1](http://uet.vnu.edu.vn/~tantd/DSP/FreqResp.pdf), [2](https://cet.ctu.edu.vn/images/upload/daotao/decuong/KC101.pdf))

@mlbvn/handson-ml

**Comment by thanhcsf** [comment]
Phản ứng hay dùng bên Y sinh, các ref em đưa sao họ chọn dùng đáp ứng nhỉ? Mình vẫn đang nghiêng về phản ứng.

**Reply by minhduc0711** [reply]
Em cũng không hiểu, đáp ứng nghe như kiểu "satisfy" :confused:

**Comment by cuongvng** [comment]
hm em vote "đáp ứng", học signal proc nghe quen vậy rồi.

**Comment by thanhcsf** [comment, marked as answer]
Anh đi một vòng chưa tìm được tài liệu nào lâu đời đề cập từ đáp ứng, dùng thì lại khá nhiều. Mặc dù anh và Đức có vẻ thấy từ phản ứng smooth hơn, nhưng tạm chốt chỗ này là đáp ứng cho phù hợp.

Nếu có thông tin phản biện khác thì mình cập nhật từ này sau nhé.
